Modulefile

```diff
@@ -7,6 +7,6 @@ summary 'Minecraft / CraftBukkit Server v2'
 description 'Install and manage a Minecraft or CraftBukkit server'
 project_page 'https://github.com/andschwa/puppet-minecraft'
 
-dependency 'maestrodev/wget', '>= 1.1.0'
+dependency 'puppet/archive', '>= 0.5.1'
```

Comment:
Are there any significant changes between archive 0.5.x and 1.0.0 that we need to be aware of outside Ruby 1.8 which I believe is now unsupported by everyone.